docs/python/django/README.md

```diff
@@ -0,0 +1,314 @@
+urlpatterns = [
+    path('', views.index, name='index'),
+]
+```
+```python
+# polls/views.py
+
+from django.http import HttpResponse
+from .models import Question
+
+def index(request):
+    count = Question.objects.count()
+    return HttpResponse(f"There are {count} questions in the DB.\n")
+```
+#### With DB Driver
+```python
+# settings.py
+
+MIDDLEWARE = [
+    'sqlcommenter.django.middleware.SqlCommenter',
+    ...
+]
+
+SQLCOMMENTER_WITH_CONTROLLER = False
+SQLCOMMENTER_WITH_FRAMEWORK = False
+SQLCOMMENTER_WITH_ROUTE = False
+SQLCOMMENTER_WITH_DB_DRIVER = True
+```
+```python
+# polls/urls.py
+
+from django.urls import path
+from . import views
+
+urlpatterns = [
+    path('', views.index, name='index'),
+]
+```
+```python
+# polls/views.py
+
+from django.http import HttpResponse
+from .models import Question
+
+def index(request):
+    count = Question.objects.count()
+    return HttpResponse(f"There are {count} questions in the DB.\n")
+```
+From the command line, we run the django development server in one terminal:
+```
+python manage.py runserver
+```
+And we use [curl](https://curl.haxx.se/) to make an HTTP request in another:
+```
+curl http://127.0.0.1:8000/polls/
+```
+### Results
+
+Examining our Postgresql server logs, with the various options
+
+#### Defaults
+```
+2019-07-19 14:27:51.370 -03 [41382] LOG:  statement: SELECT COUNT(*) AS "__count" FROM "polls_question"
+/*controller='index',framework='django%3A2.2.3',route='polls/'*/
+```
+#### With OpenCensus
+```
+2019-07-19 17:39:27.430 -03 [46170] LOG:  statement: SELECT COUNT(*) AS "__count" FROM "polls_question"
+/*traceparent='00-fd720cffceba94bbf75940ff3caaf3cc-4fd1a2bdacf56388-01'*/
+```
+
+#### With App Name
+```
+2019-07-19 15:31:33.681 -03 [42962] LOG:  statement: SELECT COUNT(*) AS "__count" FROM "polls_question"
+/*app_name='polls'*/
+```
+#### With DB Driver
+```
+2019-07-19 14:47:53.066 -03 [41602] LOG:  statement: SELECT COUNT(*) AS "__count" FROM "polls_question"
+/*db_driver='django.db.backends.postgresql'*/
+```
+## References 
+
+| Resource               | URL                                                                                                   |
+| ---------------------- | ----------------------------------------------------------------------------------------------------- |
+| Django                 | https://www.djangoproject.com/                                                                        |
+| OpenCensus             | https://opencensus.io/                                                                                |
+| opencensus-ext-django  | https://github.com/census-instrumentation/opencensus-python/tree/master/contrib/opencensus-ext-django |
+| sqlcommenter on PyPi   | https://pypi.org/project/google-cloud-sqlcommenter                                                    |
+| sqlcommenter on Github | https://github.com/open-telemetry/opentelemetry-sqlcommenter                                                                |
```

Comment:
Maybe we should add link the urls: Does this work: <https://www.markdownguide.org/basic-syntax/#urls-and-email-addresses>